department-of-veterans-affairs/va.gov-cms
# VACMS-7746 updated claro color tokens

## Description

Relates to #7746. Builds on the text updates in PR #7831. This imports all of the updated token colors from figma into our theme, and updates any existing references to use the new values. Updated color tokens seen here: https://www.figma.com/file/ScABI31f3MRFEvQ8WitVPa/Design-System-Draft?node-id=0%3A1

Also contains various tweaks to other things, per @cjterdi.

- Removes box shadow on buttons inside tables. no fuzzy table buttons
- Updates tabledrag colors for move & hover states
- Adds outline button style for form actions, to emphasize save button.
- Adds pinline to table header for paragraphs fields
- Fixes claro svg colors for details elements

## Testing done
- /node/43087/edit
- /node/3776/edit
- /admin/content

## Screenshots
<img width="825" alt="Screen Shot 2022-02-02 at 5 23 56 PM" src="https://user-images.githubusercontent.com/11279744/152265707-52148791-87a6-4c24-902e-54b46a8046f0.png">
<img width="799" alt="Screen Shot 2022-02-02 at 5 24 06 PM" src="https://user-images.githubusercontent.com/11279744/152265708-78faf2eb-737c-472c-961c-56b2c38d5542.png">
<img width="427" alt="Screen Shot 2022-02-02 at 5 24 18 PM" src="https://user-images.githubusercontent.com/11279744/152265710-f1ca80b5-05c2-4d12-b2c6-ed904c25ef61.png">
<img width="407" alt="Screen Shot 2022-02-02 at 5 24 26 PM" src="https://user-images.githubusercontent.com/11279744/152265711-674357b4-0011-4c1d-a649-a87ce864904c.png">


## QA steps

As user, log in and see that the colors have been updated slightly. ha.
- Visit `/node/3776/edit` and move paragraphs around in the table. Notice the yellows feel consistent with other VA CMS styles.
- Look at the bottom of a node edit form, see the new button styles.
- Look at details elements, either in main form or in the right rail. The icon should match the text color by default & hover.


### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
